util/mkerr.pl

```diff
@@ -702,7 +702,7 @@ sub help
     open(OUT, ">$statefile.new")
         || die "Can't write $statefile.new, $!";
     print OUT <<"EOF";
-# Copyright 1999-2017 The OpenSSL Project Authors. All Rights Reserved.
+# Copyright 1999-$YEAR The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Hmmm, `util/openssl-update-copyright` will probably change that back (well, to 2018, really)...  @mspncp ?

Reply:
@levitte, I just changed it to match the format already used within the file at L441 and L546 when auto generating the .h and .c files.

Reply:
Did you note how I refered to @mspncp?  This is not a complaint about *your* change, but rather a note that `util/openssl-update-copyright` might need a fix.

Reply:
No, it's not a problem. The regular expression for year_range is very restrictive, so it won't match $YEAR. See the comment before sed_script.

```
this_year="$(date '+%Y')"
some_year="[12][0-9][0-9][0-9]"
year_range="(${some_year})(-${some_year})?"

copyright_owner="The OpenSSL Project"
copyright="Copyright .*${year_range} .*${copyright_owner}"

# sed_script:
#   for all lines that contain ${copyright} : {
#     replace years yyyy-zzzz (or year yyyy) by yyyy-${this_year}
#     replace repeated years yyyy-yyyy by yyyy
#   }
sed_script="/${copyright}/{ s/${year_range}/\1-${this_year}/ ; s/(${some_year})-\1/\1/ }"
```

Reply:
2020

Reply:
@tgabb68 what do you mean with "2020"?